modules/http/src/main/java/com/spotify/ffwd/http/HttpOutputPlugin.java

```diff
@@ -79,7 +81,7 @@ public ILoadBalancer setupRibbonClient(
             protected void configure() {
                 final Key<HttpPluginSink> sinkKey =
                     Key.get(HttpPluginSink.class, Names.named("httpSink"));
-                bind(sinkKey).to(HttpPluginSink.class);
+                bind(sinkKey).to(HttpPluginSink.class).in(Scopes.SINGLETON);
```

Comment:
this limits possibility to declare multiple http output plugins

Reply:
This was a deliberate decision on the input of @udoprog. I believe that the reasoning is that there's no need to have multiple such output plugins specified.